frontend/pulse3d/src/pages/users-info.js

```diff
@@ -303,13 +343,7 @@ export default function UserInfo() {
                 <DropDownWidget
                   label="Actions"
                   options={["Delete", "Deactivate"]}
-                  disableOptions={[
-                    checkedUsers.length === 0,
-                    checkedUsers.length === 0 ||
-                      usersData
-                        .filter((user) => checkedUsers.includes(user.id))
-                        .filter((checkedUsers) => checkedUsers.suspended === "suspended").length !== 0,
-                  ]}
+                  disableOptions={[checkedUsers.length === 0, checkedUsers.length === 0]}
                   optionsTooltipText={[
                     "Must make a selection below before actions become available.",
                     "Must select a user who is active before actions become available.",
```

Comment:
Do you need the modal warning about attempting to deactivate existing   inactive users if this tooltip says must select for active users? 
Since you're removing the conditional check above ensuring they aren't suspended, I would change this tooltip message or put the disable conditional back in and remove the warning modal.

Reply:
I removed the modal and added the disable check back